Pages/CartPage/cart.css

```diff
@@ -0,0 +1,84 @@
+.cart-page-container{
+    display: flex;
+    /* column-gap: 4rem; */
+    justify-content:space-around;
+    padding:3rem;
+}
+.cart-page-header{
+    margin:4rem 0 2rem 0;
+    text-align: center;
+}
+.h-card-container{
+    display:flex;
+    max-width: 43rem;
+    min-height: 28rem;
+    box-shadow: 3px 5px 10px var(--BOX-SHADOW-LIGHTER);
+    margin-bottom: 2rem;
+} 
+.h-card-container img{
+    max-width:21rem;
+    min-height:28rem;
+    object-fit:cover;
+}
+.horizontal-cards{
+    display: flex;
+    flex-direction: column;
+    row-gap: 2rem;
+    flex:0 0 auto;
+}
+.quantity{
+    display: flex;
+    margin: 2rem 0;
+    column-gap: 1.5rem;
+}
+.plus-minus-button{
+    display:flex;
+}
```

Comment:
make a class of display flex in utils and use it wherever needed

Reply:
ok will look into it